apps/api/src/hyp3_api/api-spec/job_parameters.yml.j2

```diff
@@ -53,7 +53,7 @@ components:
           enum:
             - {{ job_type }}
         name:
-          $ref: "./openapi-spec.yml#components/schemas/name"
+          $ref: "./openapi-spec.yml#/components/schemas/name"
```

Comment:
I see the additional of this `/` in several files. Is this an update to how our yamls work?

Reply:
I believe it's just an update to how `jsonschema` expects reference paths to be written. Just a syntax update, nothing more.